src/controllers/users.ts

```diff
@@ -1,12 +1,12 @@
-import User from '../models/user';
+import models from '../models';
```

Comment:
Is good that you created an index file in the models folder, but we usually use named export instead. So in this case I would do

```suggestion
import { User } from '../models';
```

Below I left you a comment in the index.ts file of the models folder to make this possible